drivers/overlay/overlay.go

```diff
@@ -1175,19 +1192,19 @@ func (d *Driver) getLower(parent string) (string, error) {
 }
 
 func (d *Driver) dir(id string) string {
-	p, _, _ := d.dir2(id, false)
+	p, _, _, _ := d.dir2(id, false)
 	return p
 }
 
 func (d *Driver) getAllImageStores() []string {
-	additionalImageStores := d.AdditionalImageStores()
+	additionalImageStores, _ := d.AdditionalImageStores()
 	if d.imageStore != "" {
 		additionalImageStores = append([]string{d.imageStore}, additionalImageStores...)
 	}
 	return additionalImageStores
 }
 
-func (d *Driver) dir2(id string, useImageStore bool) (string, string, bool) {
+func (d *Driver) dir2(id string, useImageStore bool) (string, string, bool, bool) {
```

Comment:
(I didn’t really read the overlay implementation at all.)

Please document the meaning all (parameters and) return values of this function.